tapiriik/services/Pulsstory/pulsstory.py

```diff
@@ -161,7 +156,7 @@ def _populateActivity(self, rawRecord):
     def DownloadActivity(self, serviceRecord, activity):        
         activityID = activity.ServiceData["ActivityID"]
         if AGGRESSIVE_CACHE:
```

Comment:
Do you have any reason for maintaining this option? I honestly don't really remember why I put it in for RK (4+ years ago) and it's never once been turned on in production.